src/components/sections/collage/Collage.tsx

```diff
@@ -0,0 +1,176 @@
+import { ProductType } from "@/components/sections/popular-products/PopularProducts";
+import CollageCard from "@/components/sections/collage/CollageCard";
+
+export type CollectionType = {
+  id: number;
+  handle: string;
+  title: string;
+  images: string[];
+  type: "collection";
+  featured: boolean;
+};
+
+export type CollageItemType =
+  | CollectionType
+  | (ProductType & { type: "product"; featured: boolean });
+
+const collageItems: CollageItemType[] = [
```

Comment:
maybe move this to a const file or at the bottom of this file under the component?
so you don't have to scroll too much to get to the component.